website/docs/policy/getting-started.mdx

```diff
@@ -29,34 +28,27 @@ resources:
 ```
 
 :::info
-Enterprise customers should have access to fork policy library repo into their local repositories. 
+Enterprise customers should have access to fork policy library repo into their local repositories.
 :::
 
-## Install agent on management cluster
+## Install the policy agent
+
+### Management cluster
 
 The agent comes packaged with the WGE chart. To install it you need to set the following values:
 
 - `values.policy-agent.enabled`: set to true to install the agent with WGE
 - `values.policy-agent.accountId`: organization name, used as identifier
 - `values.policy-agent.clusterId`: unique identifier for the cluster
 
-## Install Policy Profile
-To install the policy profile on a cluster, you should select the `weave-policy-agent` from the profiles dropdown in the `Create Cluster` page. 
+### Leaf cluster
+
+To install the policy agent on a leaf cluster, you should select the `weave-policy-agent` from the profiles dropdown in the `Create Cluster` page.
```

Comment:
is there a CLI way to do this? (ie: a way to generate the yaml to add/push to my gitops repo via the command line?)

Reply:
not that I know of, but let me ask around